ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/SpacingInUnaryExpressionRule.kt

```diff
@@ -0,0 +1,33 @@
+package com.pinterest.ktlint.ruleset.standard
+
+import com.pinterest.ktlint.core.Rule
+import com.pinterest.ktlint.core.ast.ElementType
+import com.pinterest.ktlint.core.ast.children
+import com.pinterest.ktlint.core.ast.isPartOfComment
+import com.pinterest.ktlint.core.ast.isWhiteSpace
+import org.jetbrains.kotlin.com.intellij.lang.ASTNode
+
+class SpacingInUnaryExpressionRule : Rule("unary-op-spacing") {
```

Comment:
Same as in the previous PR for this rule - could you add a short comment summarizing what the rule does and, perhaps, some link to either kotlin or google styleguide if there's a mention of this.